### ISSUE TITLE
Persistence and BDK 1.0

### ISSUE THREAD

**Issue by josediegorobles** [bitmask-stack/bitmask-core#201]
In the alpha version of bdk 1.0 we see that they have new options to persistence the wallet data in a file. It's here:

https://docs.rs/bdk_file_store/latest/bdk_file_store/

We need that a lot, the extension is slow in mainnet because every time the wallet is initiated it must sync again...

So we must: 

- Watch when the bdk 1.0 is ready for production
- Prepare bitmask-core to be compatible with this
- See if bdk_file_store is compatible with wasm
-  If is compatible, implement it, if not we need that so we must see the options and coordinate with bdk team or do our own implementation

I put medium priority because 1.0 is not ready yet but as soon as is about to be ready we need this.

**Comment by cryptoquick** [comment]
Definitely an 0.7 milestone, but it's a fantastic idea! I hope we can find a way to integrate it into Carbonado, much like we do for RGB Stock.

**Comment by josediegorobles** [comment]
Yeah, it's an issue for when we have all the functionality on production. And bdk 1.0 is in alpha anyway. We have to see integration with Carbonado of course.
